src/contracts/fund/policies/MaxPositions.test.ts

```diff
@@ -1,22 +1,24 @@
-import { Eth } from 'web3-eth';
-import { HttpProvider } from 'web3-providers';
-import { Environment } from '../../../Environment';
 import { MaxPositions } from './MaxPositions';
+import { TestEnvironment, createTestEnvironment } from '../../../utils/tests/createTestEnvironment';
+import { MaxPositionsBytecode } from '../../../abis/MaxPositions.bin';
 
-describe('AddressList', () => {
-  let environment: Environment;
+describe('MaxPositions', () => {
+  let environment: TestEnvironment;
   let maxPositions: MaxPositions;
+  let max: number;
 
-  beforeAll(() => {
-    // TODO: This should be replaced with a local ganache test environment using proper test fixtures.
-    const client = new Eth(new HttpProvider('https://mainnet.melonport.com'));
-    environment = new Environment(client);
-    maxPositions = new MaxPositions(environment, '0x136f30369c6f387e537b0920ab616240662b9125');
+  beforeAll(async () => {
+    environment = await createTestEnvironment();
+
+    max = 10;
```

Comment:
Why initialize that here? Just set it on line 8 in describe() direclty if it doesn't change (as a `const`)